docs/user/unattended/index.md

```diff
@@ -4,118 +4,33 @@ sidebar_position: 2
 
 # Unattended installation
 
-One of the main use cases of Agama is unatteded installation. The user provides a file, known as a
-"profile", that describes how the system should look like (partitioning, networking, software
-selection, etc.) and Agama takes care of installing the system according to such a description. This
-approach might sound familiar to AutoYaST users.
+Agama is able to install a system without user interaction. The user provides a definition of the
+system, known as a "profile", that describes how the system should look like (partitioning,
+networking, software selection, etc.) and Agama takes care of installing the system. This approach
+might sound familiar to AutoYaST users.
 
-Although Agama defines its own [profile
-format](https://github.com/openSUSE/agama/blob/master/rust/agama-lib/share/profile.schema.json), it
-is able to partially handle AutoYaST profiles. Please, check the [AutoYaST support
-section](./autoyast) for further information.
+Agama uses [Jsonnet](https://jsonnet.org/), a superset of JSON, which allows writing readable and
+concise profiles. The example below instructs Agama to install _Tumbleweed_ and create a first user
+so you can log in.
 
-## Profile format
-
-A profile defines which options to use during installation: which product to install, localization
-settings, partitioning schema, etc. Although it sounds similar to AutoYaST, there are some essential
-differences:
-
-- Profiles are written in [Jsonnet](https://jsonnet.org/) instead of XML. Jsonnet is a superset of
-  JSON (so you can use just plain JSON), which allows for writing more readable and concise
-  profiles.
-- You can take advantage of Jsonnet to build dynamic profiles, without having to rely on _rules_ or
-  _Embedded Ruby (ERB)_. Agama injects hardware information that can be processed using the [Jsonnet
-  standard library](https://jsonnet.org/ref/stdlib.html).
-
-You can check the [Tumbleweed installation
-profile](https://github.com/openSUSE/agama/blob/master/rust/agama-lib/share/examples/profile_tw.json)
-included in the repository to get an impression of how a profile looks like.
-
-### A minimal example
-
-```json
+```jsonnet
 {
-  "localization": {
-    "language": "en_US.UTF-8"
-  },
-  "product": {
-    "id": "Tumbleweed"
+  product: {
+    id: "Tumbleweed"
   },
-  "user": {
-    "fullName": "Jane Doe",
-    "userName": "jane.doe",
-    "password": "123456"
+  user: {
+    fullName: "Jane Doe",
+    userName: "jane.doe",
+    password: "123456"
```

Comment:
The docs is fine but let me take this opportunity to show that while I am validating all the examples in this PR, I have found a bug in the recent [users PR](https://github.com/agama-project/agama/pull/2461).

The first object is what I paste in, the second one is agama output:
```console
cc153869b9da:/checkout # agama config generate -
{
  user: {
    fullName: "Jane Doe",
    userName: "jane.doe",
    password: "123456"
  }
}

{
  "user": {
    "fullName": "Jane Doe",
    "userName": "jane.doe"
  }
}
✗ The profile is not valid. Please, check the following errors:

        * "password" is a required property. /user

✗ Internal error: the profile was made invalid by InstallSettings round trip
```

Including `hashedPassword: false` would hide the bug.

Reply:
OK, I will check it.